src/gui/tray/ActivityItemContent.qml

```diff
@@ -184,12 +195,13 @@ Item {
 
                 visible: root.showDismissButton && !fileDetailsButton.visible
 
-                icon.source: "image://svgimage-custom-color/clear.svg" + "/" + Style.ncTextColor
+                icon.source: "image://svgimage-custom-color/clear.svg/" + Style.ncTextColor
 
-                flat: true
                 display: Button.IconOnly
-                hoverEnabled: true
```

Comment:
why removing this ?
do not we want hover effect on this button ?

Reply:
CustomButton already handles this